README.md

```diff
@@ -69,46 +69,147 @@ fields that can be selected, so implementations must provide this out-of-band. I
-in the attributes of `properties`.
+## Include/Exclude Semantics
+
+1. If `fields` attribute is specified as an empty object (for POST requests), an empty string (for GET requests),
+or with both `include` and `exclude` set to null or an
+empty array, then the recommended behavior is to include only fields
+`type`, `stac_version`, `id`, `geometry`, `bbox`, `links`, `assets`, and `properties.datetime`; if `properties.datetime` is null, then it is recommended to include `properties.start_datetime` and `properties.end_datetime`.
+These are the default fields to ensure a valid STAC Item is returned by default.
+Implementations may choose to include other properties, e.g., `properties.created`, but the number
+of default properties fields should be kept to a minimum.
+2. If only `include` is specified, these fields should be the only fields included.
+Any additional fields provided beyond those in the `include` list should be kept
+to a minimum, as the caller has explicitly stated they do not need them.
+3. If only `exclude` is specified, the specified fields fields should not be
+included in the response Item entity, but every other field available for the
+Item should be included.
+4. If `exclude` is specified and `include` is null or an empty
+array, then the `exclude` fields should be excluded from the default set.
+5. For nested fields (e.g. `properties.datetime`), the most specific path
+should be honored first, and `include` should be preferred over `exclude`. For
+example:
+    1. If a field is in `exclude`, and a nested field of that field is in
+    `include`, the nested field should be included, but no other nested
+    fields in the field should be included.  For example, if `properties` is
+    excluded and `properties.datetime` is included, then `datetime`
+    should be the only nested field in `properties`.
+    2. If a field is in `include`, and a nested field is in `exclude`, the field
+    should be included, and the nested field should be excluded.  For example,
+    if `properties` is included and `properties.datetime` is excluded, then
+    `datetime` should not be in `properties`.
+6. If the same field is present in both `include` and `exclude`, it should be included.
+
+### `null` vs empty vs missing
+
+There is a semantic difference between a missing value (i.e. if `include` is not
+in the JSON object), and a `null` or empty value.  The recommended behavior
+around missing vs. null and empty values is described in [the section
+above](#includeexclude-semantics), but is summarized here for reference. "ALL"
+means that all of the Item's fields should be returned; "DEFAULT" means the
+default set (i.e. the required fields for a valid STAC Item) should be returned.
+
+| include | exclude | returned |
+| -- | -- | -- |
+| missing | missing | DEFAULT |
+| `null` or empty | missing | DEFAULT |
+| `null` or empty | `null` or empty | DEFAULT |
```

Comment:
this is missing `| missing | `null` or empty|` and I think all 4 cases can be reduced to missing, null, or empty for either is DEFAULT, no?